proto/osmosis/incentives/group.proto

```diff
@@ -60,4 +60,9 @@ message Group {
   uint64 group_gauge_id = 1;
   InternalGaugeInfo internal_gauge_info = 2 [ (gogoproto.nullable) = false ];
   SplittingPolicy splitting_policy = 3;
-}
\ No newline at end of file
+}
+
+// CreateGroup is called via governance to create a new group gauge. Currently,
+// this only takes an array of pool IDs, but if we extend this to external
+// gauges, this will expand to add coins and numEpochsPaidOver fields.
```

Comment:
Spec should be specific as of today. If we never extend this, this detail about the future will become confusing.

```suggestion
// CreateGroup is called via governance to create a new group.  
// It takes an array of pool IDs to split the incentives across. 
```

Reply:
https://github.com/osmosis-labs/osmosis/pull/6547/commits/cb2b2a0756528d492c159ed468ef42d763c51cb3